staging/src/k8s.io/endpointslice/util/endpoint/controller_utils.go

```diff
@@ -302,3 +302,75 @@ func stringPtrChanged(ptr1, ptr2 *string) bool {
 func EndpointSubsetsEqualIgnoreResourceVersion(subsets1, subsets2 []v1.EndpointSubset) bool {
```

Comment:
This is only used in `pkg/controller/endpoint/endpoints_controller.go`. It can be moved there and made a private function to reduce the API surface.

Reply:
done